skaffold.yaml

```diff
@@ -1190,6 +1190,17 @@ profiles:
     value:
       global:
         gardenlet:
+          imageVectorOverwrite: |
+            images:
+            - name: gardenlet
+              repository: localhost:5001/eu_gcr_io_gardener-project_gardener_gardenlet
+              tag: "{{.IMAGE_TAG6}}"
+            - name: gardener-resource-manager
+              repository: localhost:5001/eu_gcr_io_gardener-project_gardener_resource-manager
+              tag: "{{.IMAGE_TAG7}}"
+            - name: gardener-seed-admission-controller
+              repository: localhost:5001/eu_gcr_io_gardener-project_gardener_seed-admission-controller
+              tag: "{{.IMAGE_TAG8}}"
```

Comment:
Instead of defining this twice, can you rather move it after L1166?

Reply:
This does not work, because the valueTemplates would be replaced again in `ha-single-zone` and `ha-multi-zone`.
But we can put the failureTolerance definitions into own values.yaml files. In this case we could use the default imageVectorOverwrite.